src/org/loklak/harvester/Post.java

```diff
@@ -35,8 +37,16 @@ public void setTimestamp() {
         this.setTimestamp(timestamp);
     }
 
-    //public abstract void getPostId();
+    public Date getTimestampDate() {
+        return new Date(this.timestamp);
+    }
+
+    //TODO: Set up TwitterTweet before setting this as abstract
+    public void setPostId() { }
```

Comment:
Even if you set it to `abstract`, it should take some argument to which it can set the ID to.

I still don't understand how setting this `abstract` now will not work but setting it `abstract` after introducing `TwitterScraper` will work. If you don't have a unique identifier for QuoraScraper now, how will you produce it when `TwitterScraper` is introduced?

And I still think that there can be a unique identifier for each Quora profile that is scraped.